src/components/ModalSearchplace/ModalSearchPlace.js

```diff
@@ -0,0 +1,86 @@
+import { IoClose, IoSearch } from 'react-icons/io5';
+
+import styles from './ModalSearchPlace.module.css';
+
+const locationSelect = ['LonDon', 'Barcelona', 'Long Beach'];
+
+function ModalSearchPlace({ handleFindLocation, handleClose }) {
+  function getSearchedLocationItems(inputValue) {
+    const locationElements = document.querySelector('#locationList').querySelectorAll(`div[class='${styles.locationItem}']`);
+    for (let i = 0; i < locationElements.length; i++) {
+      if (locationElements[i].innerText.toLowerCase().indexOf(inputValue) > -1) {
+        locationElements[i].style.display = '';
+      } else {
+        locationElements[i].style.display = 'none';
+      }
+    }
```

Comment:
không hiểu

Reply:
So sánh `innerText` trong mỗi `locationElement` với từ nhập vào để lọc kết quả tìm kiếm. `locationElement` có `innerText` phù hợp sẽ được hiển thị và ngược lại.

Reply:
các tương tác nên tương tác với dữ liệu được lưu, thay vì là dữ liệu trên các component. 

hạn chế dùng vòng lặp for, thay vào đó dùng các hàm được cung cấp bởi Js tuỳ trường hợp như `map`, `forEach`, `reduce`